crowbar_framework/app/models/nova_service.rb

```diff
@@ -185,11 +185,14 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
     neutron = ProposalObject.find_proposal("neutron",role.default_attributes["nova"]["neutron_instance"])
 
     all_nodes.each do |n|
-      if neutron["attributes"]["neutron"]["networking_mode"] == "gre"
-        net_svc.allocate_ip "default", "os_sdn", "host", n
-      else
-        net_svc.enable_interface "default", "nova_fixed", n
-        if neutron["attributes"]["neutron"]["networking_mode"] == "vlan"
+      # TODO(toabctl): The same code is in the neutron barclamp. Should be extracted and reused!
+      #                (see crowbar_framework/app/models/neutron_service.rb)
+      if neutron["attributes"]["neutron"]["networking_plugin"] == "ml2"
+	if neutron["attributes"]["neutron"]["ml2_type_drivers"].include?("gre")
```

Comment:
> Wrong indentation. Use 2 spaces.

Done in https://github.com/crowbar/barclamp-nova/commit/a1841c17f6a764b84e31838f1b4c06f28ee887b3